src/Command/EnableMaintenanceCommand.php

```diff
@@ -38,17 +44,37 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $maintenanceConfiguration = $this->configurationFactory->get();
-        $maintenanceConfiguration->setEnabled(true);
-        /** @var array $ipsAddress */
-        $ipsAddress = $input->getArgument('ips_address');
-        if ([] !== $ipsAddress) {
-            $maintenanceConfiguration->setIpAddresses(implode(',', $ipsAddress));
-        }
-        $this->maintenanceExporter->export($maintenanceConfiguration);
+        $this->getMaintenanceConfiguration($input);
```

Comment:
I don't understand why you introduice a class variable, you just need to return `MaintenanceConfiguration` on the `getMaintenanceConfiguration()` function, no ?

Reply:
I used a class variable in precedent commit which wasn't ok, i rollback